example/app.js

```diff
@@ -95,8 +95,14 @@ class BadButton extends React.Component {
 
 const ErrorBoundary = bugsnagClient.use(bugsnag__react())
 
+// You can provide a FallbackComponent to the ErrorBoundary which will be rendered if an error is encountered
+// It will be passed the `error` and `info` from the `componentDidCatch` method as props
+const FallbackComponent = ({_error, info}) => (
```

Comment:
`_error` should be `error` (even though it's not being used here, it may be a little confusing for anybody copying/pasting the examples).